apps/accounts/serializers/account/account.py

```diff
@@ -263,6 +265,21 @@ def setup_eager_loading(cls, queryset):
         )
         return queryset
 
+    def validate(self, attrs):
+        instance = getattr(self, "instance", None)
+        if instance:
+            return super().validate(attrs)
+
+        field_errors = {}
+        for _fields in Account._meta.unique_together:
+            lookup = {field: attrs.get(field) for field in _fields}
+            if Account.objects.filter(**lookup).exists():
+                for field in _fields:
+                    verbose_name = Account._meta.get_field(field).verbose_name
+                    field_errors[field] = _(f"This {verbose_name} conflicts with an existing account.")
```

Comment:
An account with this {field_name} already exists. The combination of fields {','.join(_fields)} must be unique.